actors/init/src/state.rs

```diff
@@ -28,36 +28,58 @@ impl State {
         Ok(Self { address_map: empty_map, next_id: FIRST_NON_SINGLETON_ADDR, network_name })
     }
 
-    /// Allocates a new ID address and stores a mapping of the argument address to it.
-    /// Fails if the argument address is already present in the map to facilitate a tombstone
-    /// for when the predictable robust address generation is implemented.
-    /// Returns the newly-allocated address.
-    pub fn map_address_to_new_id<BS: Blockstore>(
+    /// Maps argument addresses to to a new or existing actor ID.
+    /// With no delegated address, or if the delegated address is not already mapped,
+    /// allocates a new ID address and maps both to it.
+    /// If the delegated address is already present, maps the robust address to that actor ID.
+    /// Fails if the robust address is already mapped, providing tombstone.
```

Comment:
@anorth I find the "providing tombstone" phrasing confusing -- I _think_ I know what it means from other context, but I was wondering if you could please elaborate / rephrase here.

Reply:
Thanks!